addon/components/base/bs-button.js

```diff
@@ -442,16 +432,17 @@ export default class Button extends Component {
    * @method click
    * @private
    */
-  click() {
-    let action = this.get('onClick');
+  @action
+  handleClick(e) {
+    let onClick = this.get('onClick');
```

Comment:
This was a weird one! This failed with some undefined error when the local variable had the same name ("action") as the (newly added) decorator. Probably the decorator babel transform produced some wrong code!? 🤔

Reply:
Wondering if we should report it upstream?

Reply:
Yeah, probably. But haven't figured out where yet. It's probably an issue upstream for the babel decorator plugin, or babel itself...